mlflow/server/js/src/utils/Utils.test.js

```diff
@@ -1,9 +1,7 @@
 import Utils from './Utils';
 import { RunInfo } from '../sdk/MlflowMessages';
-import React from 'react';
```

Comment:
Why was this changed?

Reply:
4:8  error  'React' is defined but never used  no-unused-vars

This is the error that I receive when it's there.

Reply:
That seems to be because you deleted a line I the test below. Why is that?

Reply:
Added the deleted lines. Fixed that.